### DISCUSSION TITLE
Limit restful endpoint to whitelisted domain

### DISCUSSION THREAD

**Discussion by charbs-io** [supabase/supabase#17492] [Questions]
Hi, 

I have a dead simple supabase project with a single table holding over 6000 records. 

The endpoint is used on a UI that allows any unauthenticated user to search for the records, so there's an RLS policy that allows reads from anyone.

Given the needs, i had to change the max rows (in project settings > API) from 1000 to 6500. 

I am slightly worried about this being abused and I'm wondering if there's a way I can limit my endpoint/key to only be used on authorised domains? 

Thanks

**Comment by steve-chavez** [comment]
Hello,

I suggest suscribing to https://github.com/orgs/supabase/discussions/13768#discussioncomment-5634424
